train.py

```diff
@@ -30,8 +30,17 @@ def trainer(config_params):
             config_params['input']))
         raw_text, documents = annotated_file_extractor(
             config_params['input'], config_params['umls'])
-        encoded_documents = encode_data_format(
-            documents, raw_text, config_params['umls'])
+
+        label_blacklist_file = config_params['label-blacklist-file']
```

Comment:
I think this block should go to a new function, e.g. `filter_list = load_label_blacklist(label_blacklist_file)`